contrib/actions/generate-target-filters.py

```diff
@@ -23,6 +25,13 @@
 
 _filter = dict()
 
+# INCLUDE_PATTERN matches:
+# include '...'
+# include "..."
+# include("...")
+# include('...')
+INCLUDE_PATTERN = "^ *include[\(\ ][\"'](.*)[\"']\)? *$"
```

Comment:
I propose the following pattern:
```suggestion
INCLUDE_PATTERN = "^\\s*include *\\(? *[\"']([^\"']+)[\"']"
```
- Allow any leading whitespace (in Gluon we usually use tabs)
- Allow any trailing string after the closing quote (the code might have a comment)
- Make whitespace matching between `include` and the opening quote more flexible
- Fix number of backslashes in a few places
- Only allow non-quote characters between the quotes, require at least one character
I have only tested this on a Python shell and not with the Gluon code, but it seems to work as expected.